recipe/meta.yaml

```diff
@@ -41,9 +41,11 @@ test:
 
 about:
   home: https://www.datarobot.com/
-  license: Apache-2.0
-  license_family: Apache
-  license_file: LICENSE.txt
+  license: DataRobot Tool and Utility Agreement
+  license_family: PROPRIETARY
+  license_url: https://www.datarobot.com/wp-content/uploads/2021/07/DataRobot-Tool-and-Utility-Agreement.pdf
+  license_file:
+    - LICENSE.txt
```

Comment:
FYI:  This points NOT to the LICENSE.txt in this the feedstock repo, but to LICENSE.txt in the pypi package.